group.yml

```diff
@@ -151,6 +151,24 @@ repos:
     reposync:
       enabled: false
 
+  rhel-8-rt-rpms:
+    conf:
+      baseurl:
+        x86_64: http://rhsm-pulp.corp.redhat.com/content/dist/rhel8/8/x86_64/rt/os/
+        # other arches don't exist, pointing at something that exists
+        s390x: http://rhsm-pulp.corp.redhat.com/content/dist/rhel8/8/x86_64/rt/os/
+        ppc64le: http://rhsm-pulp.corp.redhat.com/content/dist/rhel8/8/x86_64/rt/os/
+        aarch64: http://rhsm-pulp.corp.redhat.com/content/dist/rhel8/8/x86_64/rt/os/
```

Comment:
RHCOS is using http://rhsm-pulp.corp.redhat.com/content/dist/rhel8/8/x86_64/nfv/os/ for the `kernel-rt` packages and note it is only supported on `x86_64`

Reply:
rt is the canonical repo for kernel-rt, question is why RHCOS is using nfv?

Reply:
We needed to also include `kernel-rt-kvm` as part of the RT kernel extension for RHCOS

https://gitlab.cee.redhat.com/coreos/redhat-coreos/-/merge_requests/877